characters/Character.js

```diff
@@ -6,14 +6,23 @@ const Character = mongoose.Schema({
   edited: Date,
   created: Date,
   gender: String,
-  height: String,
+  height: {
+    Number,
+    validate: customHeightStringValidator,
+    msg: 'Number needs to be greater then 100cm',
+    required: true
+  },
   hair_color: String,
   skin_color: String,
   eye_color: String,
   birth_year: String,
   key: { type: Number, unique: true },
+  validate: customHeightStringValidator,
+  msg: 'Number needs to be greater then 100cm',
+  required: true,
+
   homeworld_key: Number,
-  // add homeworld field that links the character to it's planet
+  characters: [ { type: ObjectId, res: 'Character' } ],
 });
-
+const HeightModel = mongoose.model( 'height', HobbiesSchema );
```

Comment:
I am unsure on the purpose of this line here, and how HobbiesSchema fits into this project.